project-reports/2024/2024-Q2-Hyperledger-Web3j.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: 2024 Q2 Hyperledger Web3j
```

Comment:
Why are we changing the Q2 report?

Reply:
I have renamed the previous Q3 to Q2, I thought it was wrongly named